content/log/index.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Collection
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+---
+
+<div class="alert alert-info">
+Datadog's log management solution is actualy in private beta. If you'd like to apply to participate in the private beta, please fill out <a href="https://www.datadoghq.com/log-management/">this form</a>.
+</div>
+
+## Overview
+{{< img src="log/index/pipeline_sketch.png" alt="Pipelines sketch" >}}
+
+## Getting started with the Agent
+
+Collecting logs is **disabled** by default in the Datadog-Agent.
+In order to start to gather your logs you need to:
+
+1. [Install the latest Datadog Agent](https://app.datadoghq.com/account/settings#agent).
+2. Uncomment those lines in your `datadog.conf` file:
+
+{{< highlight yaml >}}
+# ====================================================================== #
+#   Logs                                                                 #
+# ====================================================================== #
+# The host of the Datadog intake server to send log data to
+log_dd_url: intake.logs.datadoghq.com
+# The port to use when submitting the log lines
+log_dd_port: 10515
+{{< /highlight >}}
+
+## Collect logs with an Integrations
+To start collecting logs for an integration, you need to add a log sections in your integration yaml file, this log section need to contain:
+
+* `type` : type of log input source (**tcp** / **udp** / **file**)
+* `port` / `path` : mandatory option to specify depending on `type`. For **tcp**/**udp** `type` you have to specify the `port`, for **file** you need to specify its `path`
+* `service` : name of the service owning the log
+* `source` : \/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/
+* `sourcecategory` : \/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/\/
+
+### Examples
+#### File log collection configuration
+If you want to gather you nginx logs for instance stored in **/var/log/access.log** and **/var/log/error.log** you would edit your `nginx.yaml` file as follow
+
+{{< highlight yaml >}}
+init_config:
+instances:
+(...)
+#Log section
+logs:
+  - type: file
+    path: /var/log/access.log
+    appname: nginx
+    source: nginx
+    sourcecategory: http_access
+    tags: env:prod 
+
+  - type: file
+    path: /var/log/error.log
+    appname: nginx
+    source: nginx
+    sourcecategory: http_error
+{{< /highlight >}}
+
+#### TCP/UDP Log collection configuration
+
+If your java application doesn't log into a file but forwards its logs on the local 10514 port in TCP edit your `jmx.yaml` file as such in order to listen to this port and forward those logs to your datadog application.:
+
+{{< highlight yaml >}}
+init_config:
+instances:
+(...)
+#Log section
+logs:
+    - type: tcp
+      port: 10514
+      appname: java
+      source: java
+
+{{< /highlight >}}
+
+## Collect log from a custom log file
```

Comment:
The title could be: `Custom logs collection`

With 2 sections inside:
- `Tail existing files`
- `Stream logs through TCP/UDP`

Reply:
👍